src/main/kotlin/lotto/dto/LottoResult.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.dto
+
+private const val MINIMUM_WINNING_COUNT = 3
+
+enum class LottoResult(
+    val matchCount: Int,
+    val winningMoney: Int
+) {
+    NO_MATCH(0, 0),
+    MATCH_ONE_NUMBERS(1, 0),
+    MATCH_TWO_NUMBERS(2, 0),
+    MATCH_THREE_NUMBERS(3, 5000),
+    MATCH_FOUR_NUMBERS(4, 50000),
+    MATCH_FIVE_NUMBERS(5, 1500000),
+    MATCH_SIX_NUMBERS(6, 2000000000);
```

Comment:
```suggestion
    MATCH_THREE_NUMBERS(3, 5_000),
    MATCH_FOUR_NUMBERS(4, 50_000),
    MATCH_FIVE_NUMBERS(5, 1_500_000),
    MATCH_SIX_NUMBERS(6, 2_000_000_000);
```
언더 스코어를 활용하여 가독성을 높일수 있을것 같아요 😄